cmd/commands.go

```diff
@@ -218,6 +218,9 @@ var (
 		migrationListCommand,
 		migrationFollowCommand,
 
+		// Log drains
+		LogDrainsListCommand,
```

Comment:
```suggestion
		logDrainsListCommand,
```

There is no reason for this function to be public